### ISSUE TITLE
Constant LogLuvEncoding removed in three r135

### ISSUE THREAD

**Issue by remmel** [marcofugaro/three-projected-material#30]
Hello,

The constant `export const LogLuvEncoding = 3003;` has been removed in three r135 : https://github.com/mrdoob/three.js/commit/1d544d0fd254775c36e222e088833a3d4b3c2220

To avoid that warning 
```
WARNING in ./node_modules/three-projected-material/build/ProjectedMaterial.module.js 79:9-29
"export 'LogLuvEncoding' (imported as 'THREE') was not found in 'three'
```

The case might be removed from https://github.com/marcofugaro/three-projected-material/blob/5e0ed6d0be30db43850384fb09acb7400a2d718f/src/three-utils.js#L62

Thanks!

**Comment by marcofugaro** [comment]
AH DAMN, thanks you for the heads up!

**Comment by remmel** [comment]
in r137 GammaEncoding, LogLuvEncoding, RGBDEncoding, RGBEEncoding, RGBM16Encoding, RGBM7Encoding has been also removed
https://github.com/mrdoob/three.js/commits/r137/src/constants.js

**Comment by marcofugaro** [comment]
Fixed in [v2.1.0](https://github.com/marcofugaro/three-projected-material/releases/tag/v2.1.0)!
